.eslintrc

```diff
@@ -25,7 +26,8 @@
     "no-mixed-requires": 0,
     "no-process-exit": 0,
     "no-warning-comments": 0,
-    "curly": 0,
+    "no-loop-func": "error",
+    "curly": ["error"],
     "no-multi-spaces": 0,
```

Comment:
You may want to add those rules.
    
    "require-await": "error",
    "no-only-tests/no-only-tests": "error",
    "no-console": "error",

Reply:
Added all but `no-only-tests/no-only-tests`, since IMO test should apply the same rules but a few.